### PR TITLE
Add readme shield + update versions

### PR DESCRIPTION
**Pull request by JalexChen**
- Added a README shield
- updated versions in the README; no variants to update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +35,7 @@ For example:
 jobs:
   build:
     docker:
-      - image: cimg/base:2020.12
+      - image: cimg/base:2022.01
```
Comment (felicianotech) [inline README.md:38]: This is the chicken and egg problem we discussed. I wouldn't use the current image here as this is the base image. I would use an older one that is much less likely to be updated, or better yet, an image digest.

**Review by felicianotech — CHANGES_REQUESTED** [review]
One comment.

File: README.md (hunk 1 above)
Comment (felicianotech) [reply README.md:38]: This is the readme, ignore my comment.

**Review by felicianotech — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (JalexChen) [reply README.md:38]: confirmed this change doesn't needed to be made

**Pull request merged**